jbpm-wb-process-runtime/jbpm-wb-process-runtime-backend/src/main/java/org/jbpm/workbench/pr/backend/server/RemoteProcessRuntimeDataServiceImpl.java

```diff
@@ -206,6 +206,8 @@ protected ProcessInstanceSummary build(ProcessInstance processInstance) {
                 processInstance.getCorrelationKey(),
                 processInstance.getParentId(),
                 null,
+                processInstance.getSlaCompliance(),
```

Comment:
Would be nice to add a test that ensures the mapping between ProcessInstance and ProcessInstanceSummary works as expected.